ladybug/src/main/java/nl/nn/ibistesttool/PipeLineSessionDebugger.java

```diff
@@ -56,6 +56,11 @@ public String getMessageId() {
 		return pipeLineSession.getMessageId();
 	}
 
+	@Override
+	public Message getMessage(String key) {
+		return pipeLineSession.getMessage(key);
+	}
```

Comment:
Is het niet makkelijker om de PipeLineSessionBase te extenden en alleen de put methode te overriden?